src/Claims.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {Currency, CurrencyLibrary} from "./types/Currency.sol";
+import {IClaims} from "./interfaces/IClaims.sol";
+
+/// An intentionally barebones balance mapping only supporting mint/burn/transfer
+contract Claims is IClaims {
+    using CurrencyLibrary for Currency;
+
+    // Mapping from Currency id to account balances
+    mapping(uint256 => mapping(address => uint256)) private balances;
```

Comment:
nit: add mapping keys
```
    mapping(uint256 currency => mapping(address account => uint256)) private balances;
```

Reply:
wait also why are currencies uint256 instead of address? think thats a holdover from erc1155 that we should remove

Reply:
lol good point I was wondering that too, yea seems like a remnant from 1155